src/BoundBox.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { useRef, useState } from 'react';
+import * as cocoSsd from '@tensorflow-models/coco-ssd';
+import '@tensorflow/tfjs';
+import './index.css';
+
+interface ImageState {
+    url: string;
+    alt: string;
+    file: File;
+}
+
+const BoundBox: React.FC = () => {
+    const fileInputRef = useRef<HTMLInputElement>(null);
+    const [imageData, setImageData] = useState<string | null>(null);
+    const [predictions, setPredictions] = useState<cocoSsd.DetectedObject[] | null>(null);
+    const [isLoading, setIsLoading] = useState<boolean>(false);
+    const [selectedImage, setSelectedImage] = useState(0);
+    const [images, setImages] = useState<ImageState[]>([]);
+    const [showBoundingBox, setShowBoundingBox] = useState(true);
```

Comment:
Now that we have the back end model integration work done, can we change this BoundBox component to take in  coordinates, confidence level and classification props. Then it should use the coordinates to render a bounding box over the image.

We will be looking to integrate this BoundBox into a results page, so here may be multiple bounding boxes for a specific image, so we need to make sure we can handle n bounding boxes per image. We might need a BoundBoxes component that renders 1 image and n BoundBox components